content/how-to/fabric/common-request-templates/index.md

```diff
@@ -0,0 +1,49 @@
+---
+permalink: common-request-templates
+audit_date:
+type: product
+title: Common request templates
+created_date: '2022-03-04'
+created_by: Asmita
+last_modified_date: '2022-03-08'
+last_modified_by: Asmita
+product: Fabric
+product_url: Fabric
+---
+# Dedicated Hosting_Table
+| Common Request (Template Name)| Category | Allowed Device Type | Minimum Devices Required |
+| ----------------------------- | -------- | ------------------- | ------------------------ |
+| Add Amor anywhere| Service / Change |    - |  - |
+| Add CPU to Server or Hypervisor | Change  | Hypervisor, Hypervisor ESXi, Server, VM | One |                          
+| Add Disk Space to Server or Hypervisor | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |  
+| Add RAM to Server to Hypervisor | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |  
+| Create New User in Active Directory Domain | Account / User Mgmt | None | None |
+| Create New User on Server                  | Account / User Mgmt | None | One |
+| Add Virtual IP Address to Load Balancer | Change | Load Balancer | One or Multiple |
+| Create new VPN user on Firewall | Change | Firewall Platform, VM | One |
+| Request Emergency Financial Accomodation | Account / Account Management | - | - |
+| Create a Cloud Ticket form a Notification | Category goes here | - | - |
+| Create a Consolidate Ticket from a Notification | Category goes here | - | - |
+| Create ticket from a Notification | Category goes here | - | - |
+| Delete Contact | Account / User Management | None| None |
+| Remove Virtual IP Address from Load Balancer | Change | Load Balancer | One |
+| Generate Certificate Signing Request (CSR)   | Change| Firewall Platform, Firewall Platform WAF, VM | One |  
+| Hypervisor ESXi Version Check | Change | Hypervisor, Hypervisor ESXi, Server, Virtual Machine for Windows | One |
+| IP Allocation                 | Change | Firewall Platform, Managed Shared Storage, Server, VM | One |
+| Local User Management | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
+| Local Security Testing | Account / User Mgtm | None | - |
+| Login Access to Server | Incident | - | - |
+| Network Troubleshooting | Change | Firewall Platform WAF, Hypervisor, Hypervisor ESXi, Load Balancer, Managed Shared Storage, RPC-V, Server, VM | - |
+| Nullroute Banned IPs    | Change| Managed Shared Storage Server | None
+| OS Troubleshooting      | Incident | - | - |
+| Performance Issue       | Incident | - | - |
+| Reset User Password on Active Directory | Account / User Mgtm | None | None |
+| Reset User Password on Server | Change | VM | One |
+| Reset VPN User Password on Firewall | Change | Firewall Platform | One or Multiple |
+| Smart Hands Request for Rackspace Coloca...| Change | Colocation | None |
+| SSL Certificate | Change | Firewall Platform, VM | - |
+| User password reset / Unlock Account | Change | - | - |
+| Usage Drop | Change | Firewall Platform, Managed Shared Storage, RPC-V, Server, VM | None |
```

Comment:
```suggestion
| Usage Drop | Change | All | None |
```
This is what the template looks like. No device selector. Only data center dropdown and then devices have to be typed in.
<img width="1209" alt="Screen Shot 2022-03-09 at 10 18 42 PM" src="https://user-images.githubusercontent.com/14296817/157588689-d0bdd60b-f987-4372-9e06-4b7b33e3c05c.png">

Reply:
@asmita32675 I assume any devices can be entered so I put `All` again. Let me know if something else would work better.